concordium-sdk/src/main/java/com/concordium/sdk/responses/identityproviders/IpDescription.java

```diff
@@ -0,0 +1,26 @@
+package com.concordium.sdk.responses.identityproviders;
+
+import lombok.Data;
+import lombok.Getter;
+
+/**
+ * Description either of an anonymity revoker or identity provider. Metadata that should be visible on the chain.
+ */
+@Data
+@Getter
+public class IpDescription {
```

Comment:
There is already a `Description` in the SDK.

Reply:
done